canonical-websites/conjure-up.io
# Upgrade to Vanilla framework v1.6.3

## Done
Updated to the latest `./run` script. Replaced `vanilla-brouchure-theme` with `vanilla-framework: 1.6.3`. Updated the patterns and settings to use Vanilla. Removed unneeded local patterns.

## QA
- Run `./run`
- Run `./run clean`
- Run `./run test`
- Run `./run watch`
- Check there are no errors
- Go to http://0.0.0.0:8005/
- Check the looks :ok:

## Issue / Card
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1463
